stdlib/public/core/Mirror.swift

```diff
@@ -660,8 +660,22 @@ internal extension Mirror {
 
 //===--- QuickLooks -------------------------------------------------------===//
 
-/// The sum of types that can be used as a Quick Look representation.
```

Comment:
Let's keep the old abstract here and in the next one—the `@availability` deprecation gives enough information to mark the symbol.